aim/db/models.py

```diff
@@ -32,6 +32,13 @@ class Tenant(model_base.Base, model_base.HasDisplayName,
                       model_base.to_tuple(model_base.Base.__table_args__))
 
 
+class Infra(model_base.Base, model_base.AttributeMixin,
+            model_base.HasAimId, model_base.HasName):
+    __tablename__ = 'aim_infra_netflow'
```

Comment:
This should be just aim_infra as its not specific to netflow.

Reply:
addressed